src/algorithms/crypto/hash.go

```diff
@@ -0,0 +1,11 @@
+package crypto
+
+import (
+	util "github.com/filecoin-project/specs/util"
+)
+
+type Bytes = util.Bytes
+
+func SHA256(Bytes) Bytes {
```

Comment:
There's an impl in the filproofs lib (algorithms.go) but it causes some import cycles so this is fine. May be worth moving theirs here and having them import it (with Porcu's consent)

Reply:
OK -- am inclined to leave as is for merge velocity, but we should definitely coalesce the various impls soon (I think there are other stub impls throughout the codebase as well).